src/api/spec/services/trigger_controller_service/scm_extractor_spec.rb

```diff
@@ -95,8 +53,10 @@
         }
       end
 
-      it 'returns a hash with the extracted data from the github payload' do
-        expect(subject.call).to include(expected_hash)
+      it 'returns an instance of ScmWebhook with the extracted data from the GitHub payload' do
+        scm_webhook = subject.call
```

Comment:
I just double-checked, because I wasn't sure anymore, [`subject` is memoized within an example](https://relishapp.com/rspec/rspec-core/v/3-10/docs/subject/explicit-subject#the-%60subject%60-is-memoized-within-an-example-but-not-across-examples), so I can simply use the subject multiple times in an example.

Reply:
Please have another look.